utils/python/CIME/case_st_archive.py

```diff
@@ -180,7 +181,7 @@ def get_histfiles_for_restarts(case, archive, archive_entry, restfile):
         cmd = "ncdump -v %s %s " %(rest_hist_varname, os.path.join(rundir, restfile))
         rc, out, error = run_cmd(cmd)
```

Comment:
This doesn't look right to me.  The error isn't going to be reported under normal run conditions, is that okay?